Active Tab bug

pimber opened the pull request:
Active Tab bug er fikset og ved åbning af applikationen starter man på firste side som er home page
Credits to Loke for hjælpen.

pimber committed:
Active Tab bug

Active Tab bug er fikset og ved åbning af applikationen starter man på firste side som er home page
Credits to Loke for hjælpen.

jonasKjellerup commented on Flea/Pages/Lopper.razor:
```diff
@@ -0,0 +1,6 @@
+@page "/Loppe"
```
File and `@page` name should be adjusted to match the internal naming convention (e.g. it should match the name of the model)

pimber replied:
Done

jonasKjellerup commented on Flea/Shared/NavTab.razor:
```diff
@@ -1,9 +1,11 @@
+@using System.Security.Cryptography.X509Certificates
+@using System.Runtime.CompilerServices
```
Unused imports (`@using` directives) should be removed. (Not really that important)

pimber replied:
Deleted

jonasKjellerup commented on Flea/Pages/Marked.razor:
```diff
@@ -0,0 +1,6 @@
+@page "/Marked"
```
Again file and `@page` names should be consistent with models.

pimber replied:
Done

jonasKjellerup commented on Flea/Shared/MainLayout.razor.css:
```diff
@@ -5,7 +5,8 @@
 .main {
-    flex: 1;
+    background-color: #0d0d0d;
+    width: -webkit-fill-available;
```
Probably not the solution you are looking for.
You generally want to avoid (or at least provide fallback values for) vendor prefixed (e.g. `-webkit-*`, `-moz-*`, etc.) css properties/values since they have very limited browser compatibility and sometimes do not conform to standards.

(This example in particular does not work in Firefox 😉)


`flex:1;` with undefined `width` is generally preferred (since the surrounding container is a flexbox), but I assume that it was causing some issue since you changed it?

Otherwise you can try `width: calc(100vw - 64px);`.

pimber replied:
Fixed

jonasKjellerup commented on Flea/Shared/NavMenu.razor:
```diff
@@ -1,16 +1,19 @@
-﻿<!-- Flea logo -->
+﻿@inject NavigationManager _navigationManager
+@using Flea.Pages
+@using System.Threading.Tasks.Dataflow
```
(See comment for `NavTab.razor`)

pimber replied:
Done

jonasKjellerup reviewed (changes requested):
Good work fixing the bug. There are however a few minor issues with some of the changes that should be fixed before merge.

pimber committed:
BugActiveTab - rettelser

- Rettet @page name til det samme som file name
- Slettet udnøvendige @using
- Rettet background color, så det burde virke på FireFox

jonasKjellerup committed:
Resolved remaining requests for PR #20
